package.json

```diff
@@ -48,15 +25,70 @@
     "multiformats": "^9.4.2"
   },
   "devDependencies": {
-    "aegir": "^34.0.2",
+    "aegir": "^35.0.0",
     "util": "^0.12.4"
   },
   "eslintConfig": {
     "extends": "ipfs",
+    "parserOptions": {
+      "sourceType": "module"
+    },
     "ignorePatterns": [
       "!.aegir.js"
     ]
   },
+  "typesVersions": {
+    "*": {
+      "*": [
+        "types/src",
+        "types/src/*"
+      ]
+    }
+  },
+  "exports": {
+    ".": {
+      "import": "./src/index.js"
+    },
+    "./compare": {
+      "import": "./src/compare.js"
+    },
+    "./concat": {
+      "import": "./src/concat.js"
+    },
+    "./equals": {
+      "import": "./src/equals.js"
+    },
+    "./from-string": {
+      "import": "./src/from-string.js"
+    },
+    "./to-string": {
+      "import": "./src/to-string.js"
+    },
+    "./xor": {
+      "import": "./src/xor.js"
+    },
+    "./index.js": {
+      "import": "./src/index.js"
+    },
+    "./compare.js": {
+      "import": "./src/compare.js"
+    },
+    "./concat.js": {
+      "import": "./src/concat.js"
+    },
+    "./equals.js": {
+      "import": "./src/equals.js"
+    },
+    "./from-string.js": {
+      "import": "./src/from-string.js"
+    },
+    "./to-string.js": {
+      "import": "./src/to-string.js"
+    },
+    "./xor.js": {
+      "import": "./src/xor.js"
+    }
```

Comment:
Why do we need to double up these exports?  Could be quite error prone over time, maybe we could get ipjs to do this for us?

Reply:
We talked about this offline a couple of weeks. Anyway, we do not need this anymore. Removed now